I see that the error is due to

```
2026-01-26 11:26:30,614 334 INFO odoo odoo.addons.base_geolocalize.models.base_geocoder: openstreetmap nominatim service called 
2026-01-26 11:26:30,615 334 WARNING odoo odoo.addons.base_geolocalize.models.base_geocoder: Request to openstreetmap failed.
Code: 403
Content: b'<html>\n<head>\n<title>Access blocked</title>\n</head>\n<body>\n<h1>Access blocked</h1>\n\n<p>You have been blocked because you have violated the\n<a href="https://operations.osmfoundation.org/policies/nominatim/">usage policy</a>\nof OSM\'s Nominatim geocoding service. Please be aware that OSM\'s resources are\nlimited and shared between many users. The usage policy is there to ensure that\nthe service remains usable for everybody.</p>\n\n<p>Please review the terms and make sure that your\nsoftware adheres to the terms. You should in particular verify that you have set a\n<b>custom HTTP referrer or HTTP user agent</b> that identifies your application, and\nthat you are not overusing the service with massive bulk requests.\nIf you do that then the block usually goes away after some hours of inactivity.</p>\n\n<p>Please be aware that we are providing this service as-is and without any\nguarantees. We cannot provide support. If you need that, please have a look\nat comercial alternatives (also listed on the usage policy page) or set up\nyour own instance of Nominatim.</p>\n</body>\n</head>\n' 
2026-01-26 11:26:30,617 334 INFO odoo odoo.addons.geoengine_base_geolocalize.tests.test_geoengine_partner: ====================================================================== 
2026-01-26 11:26:30,617 334 ERROR odoo odoo.addons.geoengine_base_geolocalize.tests.test_geoengine_partner: ERROR: TestGeoenginePartner.test_geo_localize
Traceback (most recent call last):
  File "/opt/odoo/addons/base_geolocalize/models/base_geocoder.py", line 95, in _call_openstreetmap
    result = response.json()
  File "/opt/odoo-venv/lib/python3.10/site-packages/requests/models.py", line 900, in json
    return complexjson.loads(self.text, **kwargs)
  File "/opt/odoo-venv/lib/python3.10/site-packages/simplejson/__init__.py", line 514, in loads
    return _default_decoder.decode(s)
  File "/opt/odoo-venv/lib/python3.10/site-packages/simplejson/decoder.py", line 386, in decode
    obj, end = self.raw_decode(s)
  File "/opt/odoo-venv/lib/python3.10/site-packages/simplejson/decoder.py", line 416, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

we may be hitting openstreetmaps a bit too often? We may have to refactor the tests to not call OSM, or skip some tests in case of `Access Denied`

--- a/geoengine_base_geolocalize/tests/test_geoengine_partner.py
+++ b/geoengine_base_geolocalize/tests/test_geoengine_partner.py
@@ -1,5 +1,7 @@
 # Copyright 2015-2017 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from unittest.mock import MagicMock, patch
+
 from odoo.tests.common import TransactionCase
 
 
@@ -18,7 +20,14 @@ class TestGeoenginePartner(TransactionCase):
         partner_id.partner_longitude = 20
         self.assertTrue(partner_id.geo_point, "Should have geo_point")
 
-    def test_geo_localize(self):
+    @patch("requests.get")
+    def test_geo_localize(self, requests_get):
+        mock_response = MagicMock()
+        mock_response.status_code = 200
+        mock_response.json = MagicMock(
+            return_value=[{"lat": "49.9549071", "lon": "5.4085830"}]
+        )
+        requests_get.return_value = mock_response
         vals = {
             "name": "Partner Project",
             "street": "Rue au bois la dame",

